todo_app/templates/index.html

```diff
@@ -10,8 +10,24 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
+      <form action="/add_item" method="post">
+        <input class="form-control form-control-lg" type="text" name="title" id="title" placeholder="What do you want to do?" required>
+        <label for="#input"><em>Press <code>Return</code> to save</em></label>
```

Comment:
The `for` attribute is to bind the label to a specific input. It would mean that clicking on the label focuses on the correct input element. You do this by adding e.g. `for="title"` (matching the `id` of the input)